msg/en_US/utils.msg

```diff
@@ -1274,9 +1274,13 @@ $set 58 MSGCAT_UTIL_SET_FLASHBACK
 12 Transaction ID should be entered in %1$d seconds. The timeout can be adjusted through the 'flashback_timeout' setting in 'cubrid.conf'\n
 13 Flashback is already running. Flashback cannot be run by two processes.\n
 14 Flashback cannot run as standalone mode.\n
+15 Length of owner_name.class_name is required to be greater than 0 or less than %1$d bytes.\n
+16 Length of class name is required to be greater than 0 or less than %1$d bytes.\n
+17 Length of owner name is required to be greater than 0 or less than %1$d bytes.\n
+18 Owner name for '%1$s' is required to be specified.\n
```

Comment:
How about reducing the number of error messages. My suggestion is the following.
- Invalid format of owner_name.class_name ('%1$s').
- The length of the owner_name must be less than %2$d and the length of the table_name must be less than %2$d.

Reply:
> Invalid format of owner_name.class_name ('%1$s').
It seems better

In the beginning, as your suggestion, I tried to reduce the number of error messages. 
I thought, however, it would be nice for the user to output an error according to each situation, so I modified it like this.

Also, the length of owner_name and class_name must both be greater than 0. 
If I put them all in one message, the message will be too long and the meaning will be unclear.

Reply:
I will change it following your suggestion.